### PR TITLE
AT_20.12.01 | <Dashboard> Jenkins Table: Verify Dropdown menu of Orga…

### PR DESCRIPTION
**Pull request by lina2989**
https://trello.com/c/iClLSOpq

### CODE REVIEW

File: cypress/e2e/tests/dashboard.cy.js [hunk 1]
```diff
@@ -119,5 +120,15 @@ describe('dashboard', () => {
             .should('deep.equal', folderPageData.folderDropdownItems)
        });
 
+       it('AT_20.12.01 | <Dashboard> Jenkins Table: Verify Dropdown menu of Organization Project name', () => {
+        cy.createOrgFolderProject(newItemPageData.folderName)
```
Comment (Rom-T) [inline cypress/e2e/tests/dashboard.cy.js:124]: You have a test for Org folder, and you create a regular folder and check its drop-down list

Comment (lina2989) [reply cypress/e2e/tests/dashboard.cy.js]: @Rom-T I don't understand? Could you please explain me:)

Comment (Rom-T) [reply cypress/e2e/tests/dashboard.cy.js]: У вас тест должен быть написан на OrgFolder, а вы его сделали на Folder. И после последнего коммита и вовсе удалили тест.

**Comment by IrynaKolh** [comment]
I closed this PR because there is no code with test and you didn't fix PR.
if you will have a questions text me in slack @Iryna K

**Pull request closed without merging**